### PR TITLE
Implemented Healthcheck based on existing health endpoint in Dockerfi…

### PR DESCRIPTION
**Pull request by pgerhard**
Pull request for https://github.com/djfarrelly/MailDev/issues/264

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,3 +12,4 @@ settings.json
 
 # Ignore npm debug log
 npm-debug.log
+.idea
```
[the remark this answers is not in the document]
Comment (pgerhard) [reply .gitignore]: Sure, however I would like to point that the gitignore already contains an entry Sublime Text (see line 10 in the gitignore). 
Considering this I would suggest leaving the ignore for the .idea directory in the file, it does'nt cause any harm but prevents someone from accidentally committing their IntelliJ / Webstorm IDE settings

File: Dockerfile [hunk 2]
```diff
@@ -3,6 +3,9 @@ MAINTAINER "Dan Farrelly <daniel.j.farrelly@gmail.com>"
 
 ENV NODE_ENV production
 
+RUN apk add --no-cache curl && \
```
[the remark this answers is not in the document]
Comment (pgerhard) [reply Dockerfile]: I changed this as suggested

File: .gitignore (hunk 1 above)
[the remark this answers is not in the document]
Comment (oktapodia) [reply .gitignore]: Good catch! Feel free to remove the sublime text as well :)

[the remark this answers is not in the document]
Comment (pgerhard) [reply .gitignore]: :)
Ok, I removed both lines

**Comment by oktapodia** [comment]
Thank you!

**Pull request merged**